src/intel_rdt.c

```diff
@@ -164,6 +166,18 @@ static void rdt_submit(const struct pqos_mon_data *group) {
   }
 #endif
 
+  if (events & PQOS_PERF_EVENT_LLC_MISS) {
+#if PQOS_VERSION >= 40400
+    uint64_t value;
+
+    int ret = pqos_mon_get_value(group, PQOS_PERF_EVENT_LLC_MISS, &value, NULL);
+    if (ret == PQOS_RETVAL_OK)
+      rdt_submit_gauge(desc, "bytes", "llc_miss", value);
```

Comment:
Hm.  I've never used collectd with derived/delta metrics, only gauges & counters.    Do collectd write plugins delta values to totals?

Otherwise there's a problem...

Write plugins do not know whether same client queried the value earlier, so they could report the latest available value on every query, meaning that one cannot add them up in case one got same metric multiple times.  Or if plugin reports given metric only once, and multiple clients are querying for them, then some of the delta values are missed...

@octo Any advice on using derive delta vs. total counter values for metrics?